supervisor/backups/manager.py

```diff
@@ -316,6 +316,14 @@ async def do_restore_full(self, backup: Backup, password=None):
             _LOGGER.error("Invalid password for backup %s", backup.slug)
             return False
 
+        if backup.supervisor_version > self.sys_supervisor.version:
+            _LOGGER.error(
+                "Backup was made on supervisor version %s, can't restore on %s. Must update supervisor first.",
+                backup.supervisor_version,
+                self.sys_supervisor.version,
+            )
+            return False
```

Comment:
backups are not full ported to the exception error handling. You can just use the new raise/log style

Reply:
I thought that was weird that it returned false here. I'm not going to do that change-over as part of this PR but I'll add it to the list. Seems pretty straightforward 👍